whyis/test/api_test_case.py

```diff
@@ -2,4 +2,29 @@
 
 
 class ApiTestCase(TestCase):
-    pass
+
+    def publish_nanopub(self, data, content_type, expected_response=201, login=True):
+        if login:
+            self.login(*self.create_user("user@example.com", "password"))
+
+        response = self.client.post("/pub", data=data,
+                                    content_type=content_type,
+                                    follow_redirects=True)
+        self.assertStatus(response, expected_response,
+                          "Expected {}, got {} as reponse".format(expected_response, response))
+
+        return response
+        
+    def get_view(self, uri, view=None, expected_mime=None, expected_template=None):
+        content = self.client.get("/about",
+                                  query_string={ a: b for a, b in zip(["uri", "view"], [uri, view]) if b is not None },
+                                  follow_redirects=True)
+        
+        if expected_template is not None:
+            self.assertTemplateUsed(expected_template)
+
+        if expected_mime is not None:
```

Comment:
The MIME type should probably be required, or?

Reply:
By default, it serves linked data on the entity. If HTML is Accept-able, then the view itself is rendered.